docs/errors/details/_err_ngrok_502.md

```diff
@@ -4,6 +4,5 @@ ngrok resource limits are per-user, not per-account. For example, an account wit
 
 ### Possible Solutions
 
-You can invite more users to join your ngrok account.
-
-If you need a more scalable option, contact sales [here](https://ngrok.com/enterprise/contact).
+- You can [invite more users to join your ngrok account](/docs/iam/users/#invitations).
+- If you need a more scalable option, [contact sales](https://ngrok.com/enterprise/contact).
```

Comment:
This could probably be rephrased to point them to one of the paid plans on ngrok.com/pricing. fyi @samcrichard

Reply:
Could suggest pay-go here.

Reply:
I reworded it and added a link to pricing 👍